php/class-ee-site.php

```diff
@@ -417,16 +428,15 @@ public function le( $args = [], $assoc_args = [] ) {
 	private function populate_site_info( $args ) {
 
 		$this->site['name'] = EE\Utils\remove_trailing_slash( $args[0] );
+		$site = Site::find( $this->site['name'] );
+		if ( $site ) {
 
-		if ( EE::db()::site_in_db( $this->site['name'] ) ) {
-
-			$db_select = EE::db()::select( [], [ 'sitename' => $this->site['name'] ], 'sites', 1 );
-
-			$this->site['type'] = $db_select['site_type'];
-			$this->site['root'] = $db_select['site_path'];
-			$this->le           = $db_select['is_ssl'];
-			$this->wildcard     = $db_select['is_ssl_wildcard'];
+			$db_select = $site->site_url;
 
+			$this->site['type'] = $site->site_type;
+			$this->site['root'] = $site->site_fs_path;
+			$this->le = null !== $site->site_ssl;
```

Comment:
please add a bracket here: 
$this->le = ( null !== $site->site_ssl );

At first look it seems we have assign this.... `$this->le = null`